packages/ember-testing/lib/events.js

```diff
@@ -88,7 +96,8 @@ function buildKeyboardEvent(type, options = {}) {
   let event;
   try {
     event = document.createEvent('KeyEvents');
-    let eventOpts = jQuery.extend({}, DEFAULT_EVENT_OPTIONS, options);
+    let eventOpts = assign({}, DEFAULT_EVENT_OPTIONS);
+    assign(eventOpts, options);
```

Comment:
`Ember.assign` is a rough polyfill for `Object.assign` and does properly accept multiple arguments.  Can you update to:

```js
let eventOpts = assign({}, DEFAULT_EVENT_OPTIONS, options);
```